network-api/networkapi/settings.py

```diff
@@ -267,14 +268,15 @@
                 'networkapi.context_processor.cloudinary',
             ])),
             'libraries': {
-                'settings_value': 'networkapi.utility.templatetags.settings_value',
-                'mini_site_tags': 'networkapi.wagtailpages.templatetags.mini_site_tags',
-                'homepage_tags': 'networkapi.wagtailpages.templatetags.homepage_tags',
+                'bg_nav_tags': 'networkapi.buyersguide.templatetags.bg_nav_tags',
                 'card_tags': 'networkapi.wagtailpages.templatetags.card_tags',
-                'primary_page_tags': 'networkapi.wagtailpages.templatetags.primary_page_tags',
+                'class_tags': 'networkapi.wagtailpages.templatetags.class_tags',
+                'homepage_tags': 'networkapi.wagtailpages.templatetags.homepage_tags',
+                'mini_site_tags': 'networkapi.wagtailpages.templatetags.mini_site_tags',
                 'multi_image_tags': 'networkapi.wagtailpages.templatetags.multi_image_tags',
                 'nav_tags': 'networkapi.utility.templatetags.nav_tags',
-                'bg_nav_tags': 'networkapi.buyersguide.templatetags.bg_nav_tags',
+                'primary_page_tags': 'networkapi.wagtailpages.templatetags.primary_page_tags',
+                'settings_value': 'networkapi.utility.templatetags.settings_value',
                 'wagtailcustom_tags': 'networkapi.wagtailcustomization.templatetags.wagtailcustom_tags',
```

Comment:
I assume changing ordering here won't have any side effect?

Reply:
Indeed, it's it's purely for humans.